config_test.go

```diff
@@ -363,3 +363,26 @@ func TestTags(t *testing.T) {
 		assert.Equal(t, "", cfg.Key)
 	})
 }
+
+func TestCustomTag(t *testing.T) {
+	s := struct {
+		Name string `custom:"name"`
+		Age  int    `custom:"age"`
+	}{}
+
+	st := store{
+		"name": "name",
+		"age":  "10",
+	}
+
+	err := confita.NewLoader(st).Load(context.Background(), &s)
+	require.NoError(t, err)
+	require.Empty(t, &s)
+
+	l := confita.NewLoader(st)
+	l.Tag = "custom"
```

Comment:
Don't you think that going for the option pattern would be beneficial here? Even if that's our only option for now, we'd have to break the API if we change something in the future.

Reply:
I've asked myself the same question but i think it would be too much even if we want to add more customization options. One reason is because the `NewLoader` already takes a variadic and the other is i don't Confita will eventually get that more complicated than it is today in terms of API capabilities.

:book: The stdlib does the same a lot and let's the user customize a type by simply assigning different values on struct fields.
 Example if we take [http.Request](https://golang.org/pkg/net/http/#Request), we have a `NewRequest()` that helps creating a request by taking params for the usual use cases, but if you want more customization you can change struct fields.